server/src/test/java/org/elasticsearch/index/engine/ReadOnlyEngineTests.java

```diff
@@ -294,4 +296,40 @@ public void testTranslogStats() throws IOException {
             }
         }
     }
+
+    public void testSearcherId() throws Exception {
+        IOUtils.close(engine, store);
+        AtomicLong globalCheckpoint = new AtomicLong(SequenceNumbers.NO_OPS_PERFORMED);
+        try (Store store = createStore()) {
+            final EngineConfig config =
+                config(defaultSettings, store, createTempDir(), NoMergePolicy.INSTANCE, null, null, globalCheckpoint::get);
+            String lastCommitId;
```

Comment:
Maybe name this `lastSearcherId`?

Reply:
Yes, see https://github.com/elastic/elasticsearch/pull/66668/commits/a6abd00ee05beb03ad281e8b87faa42e5e4e15b2